python/cudf/cudf/core/column/string.py

```diff
@@ -57,10 +64,11 @@ class StringMethods(object):
     This mimicks pandas `df.str` interface.
     """
 
-    def __init__(self, parent, index=None, name=None):
+    def __init__(self, parent, index=None, name=None, owner=None):
```

Comment:
Lets update this to follow the CategoricalAccessor:
```suggestion
    def __init__(self, column, parent=None):
```